test/test_suite.py

```diff
@@ -722,8 +722,8 @@ def _test_get_table_names(self, schema=None, table_type="table", order_by=None):
             table_names = [t for t in tables if t not in _ignore_tables]
 
             if order_by == "foreign_key":
-                answer = ["users", "user_tmp", "email_addresses", "dingalings"]
-                eq_(table_names, answer)
+                answer = {"dingalings", "email_addresses", "user_tmp", "users"}
+                eq_(set(table_names), answer)
```

Comment:
Just to clarify: the tables will be ordered by `foreign_key` instead of table name. Wouldn't this be a consistent order each time?

Reply:
Nope, some time "users" at index `0 ` and "user_tmp"  `1` or vice versa.

Reply:
Hmm, strange 🤔